src/main/java/mate/jdbc/model/Car.java

```diff
@@ -0,0 +1,67 @@
+package mate.jdbc.model;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Car {
+    private Long id;
+    private String model;
+    private Manufacturer manufacturer;
+    private List<Driver> drivers = new ArrayList<>();
```

Comment:
Consider using a more descriptive variable name for 'drivers' to clarify its purpose, such as 'assignedDrivers'